Form-Controls/styles.css

```
@@ -0,0 +1,93 @@
+html, body {
+    width: 100%;
+    height: 100%;
+    overflow-x: hidden;
+    margin: 0;
+    padding: 0;
+}
+
+body{
+    display: flex;
+    flex-direction: column;
+    font-family: Arial, sans-serif;
+    font-size: 16px;
+    color: #222;
+}
+
+header {
+    padding: 16px;
+}
+
+main{
+    flex-grow: 1;
+    padding: 16px;
+}
+
+.form {
+    width: 700px;
```

```suggestion
    max-width: 700px;
```

What if you dropped the width rule and just had max-width? What difference does this make?